### PR TITLE
DPE-1965 - feat: expose keytool app

### PR DESCRIPTION
**Pull request by marcoppenheimer**
#### `feat: expose keytool app`
- Avoids having to pull `openjdk-17-jre-headless` from `apt` as `openjdk-8-jre-headless` comes with the snap
#### `chore: switch to apache archive`
- 3.6.X is no longer available on the Apache CDN
- Still available in the archive

### CODE REVIEW

**Review by deusebio — APPROVED** [review]
LGTM

**Comment by deusebio** [comment]
Great! The CI is very very valuable! Thanks!

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+      - name: Set default config
+        run: |
+          sudo mkdir -p /var/snap/charmed-zookeeper/current/etc/zookeeper
+          sudo cp /snap/charmed-zookeeper/current/conf/zoo_sample.cfg /var/snap/charmed-zookeeper/current/etc/zookeeper/zoo.cfg
+
+      - name: Start snap services
+        run: |
+          sudo snap start charmed-zookeeper.daemon
+          sleep 5
```
Comment (zmraul) [inline .github/workflows/ci.yaml:68]: Shouldn't the snap be able to start using the default if there is no file on `/var/snap/charmed-zookeeper/current/etc/zookeeper/zoo.cfg`?

**Review by zmraul — APPROVED** [review]
Nice.

**Pull request merged**